src/web/components/App.tsx

```diff
@@ -327,7 +327,10 @@ export const App = ({ CAPI, NAV }: Props) => {
             injectPrivacySettingsLink(); // manually updates the footer DOM because it's not hydrated
             cmp.init({
                 isInUsa: countryCode === 'US',
-                pubData: { browserId: getCookie('bwid') || undefined },
+                pubData: {
+                    browserId: getCookie('bwid') || undefined,
+                    pageViewId: window.guardian.config.ophan.pageViewId,
```

Comment:
I'm not sure if this is safe or if there is a better way to access the `pageViewId`; would someone be able to advise?

Reply:
i think you might want the following:
```suggestion
                    pageViewId: window.guardian?.config?.ophan?.pageViewId,
```
at least that way you'll just get `undefined` if it fails

Reply:
@sndrs Thanks, I didn't realise chaining was available in TypeScript!